src/main/java/mate/jdbc/dao/ManufacturerDao.java

```diff
@@ -5,13 +5,13 @@
 import mate.jdbc.model.Manufacturer;
 
 public interface ManufacturerDao {
-    Manufacturer create(Manufacturer manufacturer);
+    Manufacturer create(Manufacturer driver);
```

Comment:
```suggestion
    Manufacturer create(Manufacturer manufacturer);
```
name is confusing for next people

Reply:
copy-past.
Have been fixed.